sql/moz-fx-data-shared-prod/fenix_derived/client_adclicks_history_v1/metadata.yaml

```diff
@@ -19,6 +19,7 @@ labels:
 scheduling:
   dag_name: bqetl_org_mozilla_firefox_derived
   date_partition_parameter: null
+  parameters: ["submission_date:DATE:{{ds}}"]
```

Comment:
```suggestion
```
Removing `date_partition_parameter: null` should be equivalent to adding a new parameter here?

Reply:
This table is not partitioned, but I query the new daily data and update the table

Reply:
Ah, I see 👍